src/components/CorporateCampaign/CampaignPartner.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+	<section class="campaign-partner section">
+		<div class="row">
+			<div class="small-12 medium-10 large-6 columns">
+				<img
+					v-if="heroLogo.url"
+					class="campaign-partner__img"
+					:src="heroLogo.url"
+					:alt="heroLogo.title"
+				>
+			</div>
+			<div class="small-10 large-6 columns">
+				<h2
+					v-if="headline"
+					class="campaign-partner__header"
+				>
+					{{ headline }}
+				</h2>
+				<div
+					v-if="bodyCopy"
+					class="campaign-partner__body"
+					v-html="bodyCopy"
+					ref="partnerBodyCopy"
+				></div>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+export default {
+	props: {
+		partnerAreaContent: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		heroLogo() {
+			const mediaObject = this.partnerAreaContent?.media?.[0];
+			return {
+				title: mediaObject?.title || '',
+				url: mediaObject.file?.url || ''
+			};
+		},
+		headline() {
+			return this.partnerAreaContent?.contents?.[0]?.headline || '';
+		},
+		bodyCopy() {
+			const richText = this.partnerAreaContent?.contents?.[0]?.bodyCopy;
+			return richText ? documentToHtmlString(richText) : '';
+		},
+	},
+	async mounted() {
+		await this.$nextTick();
+		// make sure all partner content links open externally
```

Comment:
Turns out Contentful's field "target" is actually just the URL. I mistook it as a window target. This function should cover it.